sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -218,6 +221,8 @@ def fetch(self, qres, path=None, error_callback=None, **kwargs):
             else:
                 local_filenames.append(name)
 
+        if path is not None:
+            path = pathlib.Path(path)
```

Comment:
Do we have a test for this as well?

Reply:
not yet - I'll add one now

Reply:
I may have missed something, but this is still a todo?

Reply:
this is now tested on `test_srs_save_path` below